mzLib/Test/TestRangeAndTolerances.cs

```diff
@@ -364,5 +365,22 @@ public void TolerancePPMGetRange()
 
             Assert.AreEqual("±1.0000 PPM", tol.ToString());
         }
+
+        [Test]
+        public void PpmToleranceWithNotch()
```

Comment:
You should add tests to make sure values that fall between the notches are excluded.